website_sale_stock_available_display/static/src/js/website_sale_stock_available_display.js

```diff
@@ -0,0 +1,12 @@
+odoo.define('website_sale_stock_available_display.load', function (require) {
+    'use strict';
+    var ajax = require('web.ajax');
+    var core = require('web.core');
+    var website_sale_stock = require('website_sale_stock.website_sale');
+    var QWeb = core.qweb;
+
+    var load_xml = ajax.loadXML('/website_sale_stock_available_display/static/src/xml/website_sale_stock_product_availability.xml', QWeb);
```

Comment:
Do you need to add this through AJAX? Or is this for doing it asynchronously?

Reply:
Yes, it is for extend website_sale_stock.product_availability template